### ISSUE TITLE
ScrAPD - Analyze traffic fatalities in Austin

### ISSUE THREAD

**Issue by rgreinho** [open-austin/project-ideas#139]
### What problem are you trying to solve?

We are trying to solve 2 problems:
1) The city does not provide a reliable data set of traffic fatalities:
    * open data: https://data.austintexas.gov/Public-Safety/2017-APD-Traffic-Fatalities/ijds-pcyq
    * APD reports: http://austintexas.gov/department/news/296
2) The users need to be able to visualize the data (i.e. with graphs or maps), and generate reports for a given period of time.

### Who will benefit (directly and indirectly) from your project?

Mostly local organizations interested in traffic safety (Pedestrian Advisory Council (PAC), Bicycle Advisory Council (BAC), Vision Zero) and anyone else looking for current human and machine readable traffic fatality data.

### What other resources/tools are currently serving the same need? How does your project set itself apart?

For existing resources, see section above.

The ScrAPD project is compose of 3 projects:
1) A tool to retrieve the data from the APD report website and convert it into various formats.
2) The data sets
3) A visualization dashboard giving the users a graphical visualization of the data they are searching.

### Where can we find any research/data available/articles?

* [2018 PAC retrospective presentation](http://www.austintexas.gov/edims/document.cfm?id=314367)
* [The Statesman](https://www.statesman.com/news/20190215/pedestrian-deaths-reached-record-levels-in-2018-police-data-show)

### What help do you need now?

The most immediate needs is to create the visualization dashboard. 

Managing the data sets would also require some help. Using the raw data to create curated/enhanced data sets would absolutely benefit the users.

The extractor is feature complete and in maintenance mode at this point.

### What are the next steps (validation, research, coding, design)?

Building a team, working on the data and display it.

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [X] Start the conversation about this idea on [Slack](https://openaustin-slackin.herokuapp.com/) Replace this link to the [#general](https://open-austin.slack.com/messages/C036FLME2) channel with your project's preferred channel.

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [x] Post an update at least once a month to this issue. Use [BASEDEF](https://github.com/open-austin/cheatsheet-basedef) for ideas, but it's ok even if your update is just "nothing new happened this month" or "we saw a small increase in traffic to our app this month". If there's no activity for two months, that's no problem, life happens. We'll just label this as `backlog` so others know you'll get back to it when you have the time. If nobody hears from you at all in more than two months, we may mark it as `abandoned` so that others can pick up this idea and run with it.
- [ ] Take 30 minutes to complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101) 
- [X] Create a [GitHub repository](https://github.com/new). Ask for help setting up permissions if you want your repository to be within the [Open Austin Github organization](https://github.com/open-austin).
- [X] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [X] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.
- [ ] Create a [team](https://help.github.com/articles/creating-a-team/) for your core contributors
  - This will make it easier for you to manage your github repo access. People on a team have the same level of access. Admin access will allow your trusted contributors to make changes as needed. 
  - You can remove and add people to your team as needed. 
  - View some of our teams: [https://github.com/orgs/open-austin/teams](https://github.com/orgs/open-austin/teams)
  - Note: You can also allow collaborators outside of your team and give them more limited access.
- [ ]   Create a [user group](https://get.slack.help/hc/en-us/articles/212906697-User-Groups) in Slack so you can "@" your core contributors all at once, without bothering other people who use the Slack channel. You'll need permission from a Slack admin, so just mention @leadership on Slack to get this set up.
- [ ]   Create a [Google Drive](https://drive.google.com), [Dropbox](https://www.dropbox.com/), or other cloud storage to share larger files. Github and Data.World are good for code and data, respectively, especially when you need version control. But they're not good for very large files, documentation, articles, etc. A cloud storage option will allow you to easily share, create, and collaborate on documents with your team and help organize ideas and thoughts.
  - Doing this early on can help your team stay organized and to onboard new contributors who wouldn't have access to files you all have shared over email.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

- [ ] In your README, link to the [Open Austin Code of Conduct](https://www.open-austin.org/about/#code-of-conduct) or write your own code of conduct.
- [X] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [X] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).
- [ ] Create an easily shareable project management artifact, like a [Civic Tech Canvas](https://cityofaustin.github.io/civic-tech-canvas/) or [Open Canvas](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/)
- [X] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [my project] to projects page`. An Open Austin leader will review this issue and post your project :balloon:
- [X] [Tell the City of Austin](http://austintexas.gov/page/mobile-app-library). If your idea is in a shareable format and can benefit people around the city, go to that site and follow the instructions on the bottom of the page to showcase your work there.

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by JMensch** [comment]
@rgreinho this looks really cool, nice work. Have you spoken to any potential end-users yet? I'm trying to wrap my head around what problems they would solve using a viz like this.

**Comment by rgreinho** [comment]
Yes I did, and this project already has potential users waiting for it: Pedestrian Advisory Council (PAC), Bicycle Advisory Council (BAC), Vision Zero.

Basically, every month they do the following:
1) Browse through http://austintexas.gov/department/news/296
2) Manually add new fatality details into a spreadsheet
3) Create a monthly report
4) At the beginning of the year they compile all the data they collected and generate a retrospective of the previous year

The goal of these projects, in addition to the ones mentioned above, is to remove the need for them to perform any manual task. Let ScrAPDviz do the work you need, stop using a spreadsheet.

**Comment by cbaumanis** [comment]
The best place to get the most accurate fatality data is from the FARS database, but it's not very up-to-date.

APD sends the crash report to TxDOT and then TxDOT uploads crash data to CRIS. CRIS shows BOTH fatalities and non-fatality crashes. There's a little lag between when the crash happens, when it gets put into CRIS, and when details get finalized. However I've personally matched CRIS to FARS and didn't find any discrepancies. CRIS public database has mapping capabilities. 

https://cris.dot.state.tx.us/public/Purchase/app/home/welcome
https://www.nhtsa.gov/research-data/fatality-analysis-reporting-system-fars

**Comment by mscarey** [comment]
I haven't compared the city's [Vision Zero dashboard](https://visionzero.austin.gov/viewer/) with ScrAPD feature-for-feature, but it might supersede this project entirely. Most impactful Code for America projects become blueprints for services provided by the government, so being superseded is a good result.

**Comment by rgreinho** [comment]
Yes, the official Vision Zero dashboard definitely supersedes this project. 

The only 2 reasons I keep this project up and running are:
1. I show the names of the victims. It is a requirement for some local organizations.
2. I check APD data hourly. The city updates the dataset on a monthly or bi-monthly basis.

**Comment by mateoclarke** [comment]
I agree that ScrAPD's project work was a useful product demonstration that helped inspire the Vision Zero Viewer team. Our team at ATD collaborated with @rgreinho during the project planning and implementation. Definitely a success in that way.
